### PR TITLE
Add new command for querying API KEY quota usage

### PR DESCRIPTION
**Pull request by Bala-SecneurX**
* Add new command to get API KEY quota usage details

Signed-off-by: Bala Manoharan <bala@secneurx.com>

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [x] Tests
- [x ] Documentation

### CODE REVIEW

**Comment by dorschw** [comment]
Hi @Bala-SecneurX, thank you for the contribution!

Reviewing this PR will be slightly delayed as we are in the midst of the local holiday season. We'll have a look as soon as we're back at work.

**Review by esharf — CHANGES_REQUESTED** [review]
Good job.
please add the new command to the `command_examples` file

File: Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py [hunk 1]
```diff
@@ -711,6 +711,9 @@ def get_report_cmd(client: Client, args: Dict[str, str]):
                 title = (f"SecneurX Analysis - Detailed Report of the Analyzed Sample: {taskUuid}")
                 readableContents = format_report_contents(contents)
                 readableOutputs = tableToMarkdown(title, readableContents, headers=headerList, headerTransform=pascalToSpace)
+                reportFileName = taskUuid + reportExtn
+                fileContent = fileResult(reportFileName, response)
+                demisto.results(fileContent)
```
[the remark this answers is not in the document]
Comment (Bala-SecneurX) [reply Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py:716]: @esharf We wanted to give the option to download the complete report JSON file to the user.

File: Packs/SecneurXAnalysis/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### SecneurX Analysis
+- Updated the Docker image to: *demisto/python3:3.10.7.35188*.
+- Adding new command for API key quota usage details.
```
[the remark this answers is not in the document]
Comment (Bala-SecneurX) [reply Packs/SecneurXAnalysis/ReleaseNotes/1_0_1.md:5]: Agreed.

File: Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (esharf) [reply Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py:716]: So please use `return_results` instead of `demisto.results`.

**Comment by Bala-SecneurX** [comment]
Thanks will update

Regards,
Bala
________________________________

**Comment by Bala-SecneurX** [comment]
@esharf I have updated all the review comments. Sorry for the delay had some issues with our cortex licensing.
Pls review the changes and provide your feedback.

**Comment by Bala-SecneurX** [comment]
@esharf Could you please review this PR

**Review by esharf — APPROVED** [review]

**Pull request merged**